pkgs/applications/networking/netperf/default.nix

```diff
@@ -1,21 +1,29 @@
-{ stdenv, fetchFromGitHub }:
+{ glibc, libsmbios, stdenv, autoreconfHook, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  name = "netperf-2.7.0";
+  name = "netperf-git";
 
   src = fetchFromGitHub {
     owner = "HewlettPackard";
     repo = "netperf";
-    rev = name;
-    sha256 = "034indn3hicwbvyzgw9f32bv2i7c5iv8b4a11imyn03pw97jzh10";
+    rev = "c0a0d9f31f9940abf375a41b43a343cdbf87caab";
+    sha256 = "0wfj9kkhar6jb5639f5wxpwsraxw4v9yzg71rsdidvj5fyncjjq2";
   };
 
+  buildInputs = [ glibc libsmbios ];
```

Comment:
Glibc shouldn’t be here. It’s provided by the compiler.

Reply:
gotcha. I see a bunch of other nix files that take glibc would it be appropriate to go and try removing them in a separate PR?

Reply:
done